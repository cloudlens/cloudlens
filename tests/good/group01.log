foo 1
foo 2
foo 3
foo 4
foo 5
foo 6
foo 7
foo 8
foo 9
foo 10
