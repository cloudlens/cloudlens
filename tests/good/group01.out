Before group: e = {"message":"foo 1","n":"1"}
Before group: e = {"message":"foo 2","n":"2","even":"true"}
Before group: e = {"message":"foo 3","n":"3"}
Before group: e = {"message":"foo 4","n":"4","even":"true"}
After group: e = {"group":[{"message":"foo 2","n":"2","even":"true"},{"message":"foo 3","n":"3"}],"message":"foo 2\nfoo 3","n":"2\n3","even":"true"}
Before group: e = {"message":"foo 5","n":"5"}
Before group: e = {"message":"foo 6","n":"6","even":"true"}
After group: e = {"group":[{"message":"foo 4","n":"4","even":"true"},{"message":"foo 5","n":"5"}],"message":"foo 4\nfoo 5","n":"4\n5","even":"true"}
Before group: e = {"message":"foo 7","n":"7"}
Before group: e = {"message":"foo 8","n":"8","even":"true"}
After group: e = {"group":[{"message":"foo 6","n":"6","even":"true"},{"message":"foo 7","n":"7"}],"message":"foo 6\nfoo 7","n":"6\n7","even":"true"}
Before group: e = {"message":"foo 9","n":"9"}
Before group: e = {"message":"foo 10","n":"10","even":"true"}
After group: e = {"group":[{"message":"foo 8","n":"8","even":"true"},{"message":"foo 9","n":"9"}],"message":"foo 8\nfoo 9","n":"8\n9","even":"true"}
After group: e = {"group":[{"message":"foo 10","n":"10","even":"true"}],"message":"foo 10","n":"10","even":"true"}
cpt = 5
